code.swift

```diff
@@ -54,26 +54,22 @@ class PeriodicTableViewController: UIViewController {
     }
     
     // This next method was added by a new member of the team. Identify the 10 formatting, naming, and logic issues below:
-    @IBAction func chooseRandomElement(_ sender: Any){
-        let ranElem = elementsContoller.elements.randomElement()
+    @IBAction func chooseRandomElement(_ sender: Any) { // added a space between ) and {
+        let randomElement = elementsContoller.elements.randomElement() // changed randElem to randomElement
         
-        var chosenElementView:ElementView!
+        var chosenElementView: ElementView! // added a space between : and E
         
-        for e in elementViews {
-            if (e.element == ranElem) {
-                chosenElementView = e
-                break;
+        for element in elementViews { // changed e to element
```

Comment:
There might be a better name that `element` here... As a tip, try to think about what type this variable has - that might help you name it!